pom.xml

```diff
@@ -86,6 +86,8 @@
                     <arguments>
                         <argument>generatePluginsMarkdown</argument>
                         <argument>${RODA_PLUGIN}</argument>
+                        <argument>[Production-ready]</argument>
```

Comment:
Add a comment stating what that argument means, in this case, that refers to the development life cycle of the plugin